static/js/cartutils.js

```diff
@@ -999,7 +999,7 @@ define(['filterutils','jquery', 'tippy', 'base' ], function(filterutils, $,  tip
                     {
                         "type": "text", "orderable": true, data: 'StudyInstanceUID', render: function (data) {
                             return pretty_print_id(data) +
-                                ' <a class="copy-this-table" role="button" content="' + data +
+                                ' <a class="copy-table" role="button" content="' + data +
```

Comment:
@G-White-ISB one small thing--I suspect this should be copy-this and not copy-table. looks good otherwise!